src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -88,9 +87,23 @@
     <Compile Include="$(CommonPath)\Interop\Windows\Crypt32\Interop.Alg.cs" >
       <Link>Common\Interop\Windows\Crypt32\Interop.Alg.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs" >
-      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\Interop.SchProtocols.cs" >
+      <Link>Common\Interop\Windows\SChannel\Interop.SchProtocols.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\Winsock\Interop.ErrorCodes.cs" >
```

Comment:
@pgavlin Given that the ErrorCodes are not conditionally compiled for Nix vs Windows, I would expect the file to be renamed and moved within Common\System\Net. 
Is this file still tightly coupled with the Winsock implementation?